src/dashboard/locale/fr/LC_MESSAGES/django.po

```diff
@@ -19,155 +19,509 @@ msgstr ""
+" Si vous avez tapé l'adresse web dans le navigateur, vérifier qu'elle est "
+"correcte. La page n'est peut-être plus disponible.<br>\n"
+"Dans ce cas, pour continuer votre visite vous pouvez consulter notre page "
+"d'accueil."
+
+#: src/dashboard/templates/500.html:5 src/dashboard/templates/500.html:14
+msgid "Unexpected error"
+msgstr "Erreur inattendue"
 
-#: templates/blocks/header.html:31
+#: src/dashboard/templates/500.html:17
+msgid "Error 500"
+msgstr "Erreur 500"
+
+#: src/dashboard/templates/500.html:20
+msgid ""
+"Oops, looks like something terrible happened. We have already been notified."
+msgstr ""
+"Désolé, le service rencontre un problème, nous travaillons pour le résoudre "
+"le plus rapidement possible."
+
+#: src/dashboard/templates/500.html:23
+msgid "Try refreshing the page or try again later."
+msgstr "Essayer de rafraichir la page ou bien ressayer plus tard."
+
+#: src/dashboard/templates/blocks/header.html:31
 msgid "Display settings"
 msgstr "Paramètres d'affichage"
 
-#: templates/blocks/header.html:50
+#: src/dashboard/templates/blocks/header.html:46
 msgid "Search"
 msgstr "Recherche"
 
-#: templates/blocks/header.html:52
+#: src/dashboard/templates/blocks/header.html:48
 msgid "Menu"
 msgstr "Menu"
 
-#: templates/blocks/language_selector.html:15
+#: src/dashboard/templates/blocks/language_selector.html:15
 msgid "Select language"
 msgstr "Sélectionner la langue"
 
-#: templates/blocks/logout.html:6
+#: src/dashboard/templates/blocks/logout.html:6
 msgid "Log Out"
 msgstr "Se déconnecter"
 
-#: templates/blocks/main_menu.html:9
+#: src/dashboard/templates/blocks/main_menu.html:9
 msgid "close"
 msgstr "fermer"
 
-#: templates/blocks/main_menu.html:13
+#: src/dashboard/templates/blocks/main_menu.html:13
 msgid "Main menu"
 msgstr "Menu principal"
 
-#: templates/blocks/main_menu.html:25
+#: src/dashboard/templates/blocks/main_menu.html:25
 msgid "Home"
 msgstr "Accueil"
 
-#: templates/registration/login.html:20
+#: src/dashboard/templates/registration/login.html:15
 msgid "Login to"
 msgstr "Se connecter"
 
-#: templates/registration/login.html:30
-msgid "Log in with your account"
-msgstr "Se connecter avec son compte"
-
-#: templates/registration/login.html:44
-msgid "Unless otherwise stated, all fields are required."
-msgstr "Sauf mention contraire, tous les champs sont obligatoires."
-
-#: templates/registration/login.html:50
-msgid "Identifier"
-msgstr "Identifiant"
-
-#: templates/registration/login.html:51
-msgid "Expected format: myusername"
-msgstr "Format attendu : nomdutilisateur"
-
-#: templates/registration/login.html:69
-msgid "Password"
-msgstr "Mot de passe"
-
-#: templates/registration/login.html:85
-msgid "Show password"
-msgstr "Afficher le mot de passe"
-
-#: templates/registration/login.html:90
-msgid "Show"
-msgstr "Afficher"
+#: src/dashboard/templates/registration/login.html:18
+msgid ""
+"\n"
+"              Description — Lorem ipsum dolor sit amet, consectetur "
+"adipiscing elit.\n"
```

Comment:
Should be fixed?

Reply:
This is a temporary text. This page is waiting for real content